docs/quickstart.md

```diff
@@ -451,3 +451,26 @@ as above:
 >>> httpx.get("https://example.com", auth=auth)
 <Response [200 OK]>
 ```
+
+## Errors and Exceptions
+
+In some situations, HTTPX can raise exceptions:
+
+* `NetworkError` in case of network problem (e.g. DNS failure, refused connection, ...)
+* `ConnectTimeout`, `ReadTimeout`, `WriteTimeout`, `PoolTimeout` in case of a configured timeout is reached. All of them inherit from `TimeoutException`
```

Comment:
```suggestion
* `ConnectTimeout`, `ReadTimeout`, `WriteTimeout`, `PoolTimeout` in case a configured timeout is reached. All of them inherit from `TimeoutException`.
```